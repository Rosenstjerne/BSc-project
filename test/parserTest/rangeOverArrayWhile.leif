var int a,b;
var int [] c;
a = 0;
b = 5;

while (a < b) {
    print(c[a]);
    a = a + 1;
}

