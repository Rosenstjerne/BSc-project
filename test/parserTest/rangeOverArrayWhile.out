function null main() {
	var int a, b;
	var int[] c;
	
	a = 0;
	b = 5;
	while((a < b)) {
	#begin while
		print((c[a]));
		a = (a + 1);
		
	}
	#end while
	
	
}
